static/templates/gear_menu.hbs

```diff
@@ -54,7 +54,7 @@
                 </a>
             </li>
             <li role="presentation">
-                <a tabindex="0" role="menuitem" data-overlay-trigger="keyboard-shortcuts">
+                <a href="#keyboard-shortcuts" tabindex="0" role="menuitem">
```

Comment:
Does removing `data-overlay-trigger` have side effects? We could just add the `a href` without removing that, if it were desired.